packages/backend-function/API.md

```diff
@@ -4,13 +4,12 @@
 
 ```ts
 
-import { Construct } from 'constructs';
 import { ConstructFactory } from '@aws-amplify/plugin-types';
 import { FunctionResources } from '@aws-amplify/plugin-types';
 import { ResourceProvider } from '@aws-amplify/plugin-types';
 
 // @public
-export const defineFunction: (props?: FunctionProps) => ConstructFactory<Construct & ResourceProvider<FunctionResources>>;
+export const defineFunction: (props?: FunctionProps) => ConstructFactory<ResourceProvider<FunctionResources>>;

```

Comment:
Here we use the ResourceProvider type because the function construct class is internal to this package and we don't want to expose it's existence in our API